encoder.py

```diff
@@ -178,20 +182,16 @@ def calc(MSG, hash_type):
         domain = string.ascii_lowercase
         positions = [domain.find(elem) for elem in key]
         lenp = len(positions)
-
```

Comment:
These four deletions have nothing to do with your additions, leave them be 🤦‍♂

Reply:
Sorry about that. After I edited them, I noticed that the specific lines you asked for were from 182 - 196. I edited those lines but forgot to set the other ones I removed back.
 
Please don't mind.